openff/interchange/_tests/energy_tests/interop/amber/test_energies.py

```diff
@@ -2,11 +2,12 @@
 from openff.toolkit import Quantity
 from openff.utilities import skip_if_missing
 
-from openff.interchange._tests import MoleculeWithConformer, requires_ambertools
+from openff.interchange._tests import MoleculeWithConformer, requires_ambertools, requires_openeye
 from openff.interchange.constants import kcal_mol
 from openff.interchange.drivers import get_amber_energies, get_openmm_energies
 
 
+@requires_openeye
```

Comment:
(not blocking) I don't follow... maybe something about the build matrix where the presence of OpenMM used to be a proxy for OE, but isn't any more?